services/ingestion/poller/cvPartner/cv_partner_process_lambda.py

```diff
@@ -0,0 +1,247 @@
+        'cv_link': 'link',
+        'cv.navn': 'navn',
+        'cv.email': 'email',
+        'cv.telefon': 'telefon',
+        'cv.born_year': 'born_year',
+        'cv.nationality.int': 'nationality',
+        'cv.place_of_residence.int': 'place_of_residence',
+        'cv.twitter': 'twitter',
+        'cv.title.no': 'title'}, inplace=True)
+
+    """
+    Transforms:
+        {
+            elem_tag: "list1[0]",
+            "cv": {
+                "topic_list": [
+                    {
+                        "topic_elem": "val0"
+                    },
+                    {
+                        "topic_elem": "val1"
+                    },
+                ]
+            }
+        },
+        {
+            elem_tag: "list1[1]",
+            "cv": {
+                "topic_list": [
+                    {
+                        "topic_elem": "val2"
+                    },
+                    {
+                        "topic_elem": "val3"
+                    },
+                ]
+            }
+        },
+    to dataframe:
+
+    |elem_tag topic_elem|
+    list1[0]  val0
+    list1[0]  val1
+    list1[1]  val2
+    list1[1]  val3
+
+    @param: elem_tag
+    @param: list1
+    @param: topic_list
+    """
+    def create_from_topic(elem_tag, list1, topic_list):
+        def create_df_for_each_elem(elem_tag, list1_elem, topic):
+            if not isinstance(topic, list):
+                return None
+            return pd.DataFrame.from_records(np.array([{elem_tag: list1_elem, **x} for x in topic]))
+        return pd.concat([create_df_for_each_elem(elem_tag,
+                         list1[indx], topic_list[indx]) for indx in range(topic_list.size)],
+                         ignore_index=True)
+
+    """
+    If a sub category of the cv structure has a dictonary as a value, it must be normalized to only
+    constitute one value in the final dataframe. The values are assumed given as
+    ....
+    sub_category : { tag1: val1, tag2: val2},
+    ....
+    and the desired output value is "norwegian translation of sub_cat value"
+
+    @param: df - original dataframe
+    @param: col_name: name of the coloumn to normalize
+    """
+
+    def normalize_coloums(df, col_name, tag1, tag2):
+        tmp_col = df[col_name].copy()
+        tmp_col = tmp_col.apply(lambda x: {} if pd.isna(x) else x)
+        tmp = pd.json_normalize(tmp_col.copy())
+        col1 = tmp[tag1].to_numpy()
+        col2 = tmp[tag2].to_numpy()
+        col1_series = pd.Series(col1, name=col_name)
+        col1_series_out = col1_series.replace(np.nan, '', regex=True)
+        col2_series = pd.Series(col2, name=col_name+tag2)
+        col2_series_out = col2_series.replace(np.nan, '', regex=True)
+        return col1_series_out, col2_series_out
+
+    def create_df(df, sub_cat, cols, normalizable_cols):
+        cat_dict = df['cv.' + sub_cat].copy()
+        user_ids = df['user_id'].copy()
+        tmp_df = create_from_topic('user_id', user_ids, cat_dict)
```

Comment:
```suggestion
        df = create_from_topic('user_id', user_ids, cat_dict)[cols].copy()
```
Så trenger vi ikke så mange `tmp*` variabler